src/progress-bar/ProgressBar.tsx

```diff
@@ -28,14 +30,16 @@ function ProgressBar(props: ProgressBarProps) {
       className={progressBarClassName}
       style={{
         backgroundColor
-      }}>
+      }}
+      data-testid={testid}>
       <div
         className={"progress-bar__track"}
         style={{
           width: `${percentage}%`,
           // eslint-disable-next-line no-magic-numbers
           backgroundColor: percentage === 100 ? completedColor : trackColor
```

Comment:
This is not related to tests but I am wondering if we should handle `percentage > 100` case for this component, user can pass a percentage that is bigger than 100, and looks like this would cause overflow (track would be too long). Maybe we can talk in our meeting

Reply:
You're right. We can change this condition like this;
```js
backgroundColor: percentage >= 100 ? completedColor : trackColor
```

Reply:
You're right. We can change like this;
```js
backgroundColor: percentage >= 100 ? completedColor : trackColor
```